onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -2802,3 +2803,270 @@ TEST(CApiTest, TestMultiStreamInferenceSimpleSSD) {
   ASSERT_TRUE(output_dims == expected_output_dims);
 }
 #endif
+
```

Comment:
could you also update the custom op example here:

[https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/test/testdata/custom_op_library](https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/test/testdata/custom_op_library)

Reply:
Sure - let's get it updated.

Reply:
custom op examples updated in custom_op_library.